src/bodies.cpp

```diff
@@ -579,76 +579,72 @@ void bodies::Box::computeBoundingCylinder(BoundingCylinder &cylinder) const
 
 bool bodies::Box::intersectsRay(const Eigen::Vector3d& origin, const Eigen::Vector3d& dir, EigenSTL::vector_Vector3d *intersections, unsigned int count) const
```

Comment:
line width too long
side note: We should add clang-format checking to this repo

Reply:
no changes by me